### DISCUSSION TITLE
Different snapshots on iOS 14.2 and 14.4 simulator

### DISCUSSION THREAD

**Discussion by haemi** [pointfreeco/swift-snapshot-testing#451] [Q&A]
When I create snapshots on an iOS 14.2 simulator, it differs a little bit from the snapshot made on an iOS 14.4 simulator (both iPhone 11, but that shouldn't matter, right?).

Sometimes the difference is much smaller than shown in the example images; sometimes it's only one letter in a paragraph...

Here are the two images and one showing the diff (made in Kaleidoscope):

<img width="414" alt="testFoo iPhoneXr__UICTContentSizeCategoryUnspecified_light" src="https://user-images.githubusercontent.com/183958/113348635-1fbe9480-9337-11eb-9b44-212147dea0cb.png">

<img width="320" alt="testFoo iPhoneSe__UICTContentSizeCategoryUnspecified_light" src="https://user-images.githubusercontent.com/183958/113348654-2816cf80-9337-11eb-8b4c-1e3621424eac.png">

<img width="322" alt="CleanShot 2021-04-01 at 22 11 16@2x" src="https://user-images.githubusercontent.com/183958/113348688-3533be80-9337-11eb-9bab-cdfb68a30f57.png">

**Comment by stephencelis** [comment]
This is expected. Apple makes small changes to how its UI renders regularly, including very minor adjustments to how fonts render. This means you'll need to re-record snapshots occasionally, and you'll need to maintain a stable environment. From our README:

> ⚠️ Warning: Snapshots must be compared using a simulator with the same OS, device gamut, and scale as the simulator that originally took the reference to avoid discrepancies between images.
